app/views/projects/show.html.haml

```diff
@@ -48,6 +48,13 @@
     = link_to 'Ruby', project_path(project, format: 'ruby'), class: 'btn btn-primary'
     = link_to 'Typescript', project_path(project, format: 'typescript'), class: 'btn btn-primary'
 
+  %h4 API Gateway Integration
+  .buttons
+    = link_to 'Swagger format',
```

Comment:
I would put your button besides Swagger one (l.36) and call it "Swagger with API Gateway integration" (and maybe show it only if project has the integration set up)

Reply:
We can generate the file even without any integration configuration so I do not think adding a condition is relevant.
I move and rename the button.

Reply:
Good point